lightning/src/ln/channelmanager.rs

```diff
@@ -539,6 +714,12 @@ impl PendingOutboundPayment {
 	}
 }
 
+pub(crate) struct ExpandedPmtKey {
```

Comment:
Could you document this? Maybe this struct should go in the `inbound_payments` module?